project/project/Common.scala

```diff
@@ -0,0 +1,84 @@
+package metabuild
```

Comment:
Copied the Common class from the build configuration to the configuration of amf's build config (recursive.) This is to access the mulesoft repository to download the plugin.